forest/src/cli/chain_cmd.rs

```diff
@@ -17,6 +22,22 @@ pub enum ChainCommands {
         cid: String,
     },
 
+    /// Export a snapshot of the chain to <output_path>
+    #[structopt(about = "Export chain snapshot to file")]
+    Export {
+        #[structopt(short, help = "Tipset to start the export from, default is @HEAD")]
+        tipset: Option<i64>,
+        #[structopt(
+            short,
+            help = "specify the number of recent state roots to include in the export"
+        )]
+        recent_stateroots: Option<i64>,
+        #[structopt(short, help = "default: false")]
```

Comment:
Would be more helpful to print a help message for the ```-s``` flag
(you can put it in a doc comment as well, so ```/// Skips old messages```).
Btw default value is expected to be false for a bool flag in a cli.

Reply:
:+1: